### PR TITLE
Front page implementation

### PR DESCRIPTION
**Pull request by amanpdyadav**
* Update user profile page to render name and address
* Add translations properties for texts
* Add address component
* Remove unnecessary types
* User translation instead of hardcoded string
* Add background to hero component
* Add social media icons to the footer of the page
* Rename parking-region-map to parking-zone-map
* Update front page to match the design
* Add feature reducers and actions
* Remove actions and reducers for zone and coordinates
* Add background to pages
* Allow front page access only by an authenticated user
* Move styles to assets folder
* Add marker icon
* Add classname package

### CODE REVIEW

File: src/i18n/fi.json [hunk 1]
```diff
@@ -1,7 +1,22 @@
...
+          "message": "Väestörekisterikeskuksesta saatujen henkilötietojesi perusteella voit ostaa pysäköintitunnuksen oheisille vyöhykkeille."
+        }
+      },
+      "selectAddress": "Valitse osoite",
+      "buyParkingId": "Osta pysäkointitunnus"
+    }
+  },
+  "common": {
+    "address": {
+      "residentParkingZone": "Asukas pysäköintialue",
```
Comment (mhieta) [inline src/i18n/fi.json:17]: This would be one word: "Asukaspysäköintialue"

File: src/i18n/fi.json [hunk 2]
```diff
@@ -1,7 +1,22 @@
 {
   "page": {
-    "front": {
-      "title": "Hei Kaisa!"
+    "frontPage": {
+      "title": "Hei Kaisa!",
```
Comment (mhieta) [inline src/i18n/fi.json:4]: As discussed earlier, this should be dynamic: "Hei <user_first_name>!"

File: src/common/navbar/Navbar.tsx [hunk 3]
```diff
@@ -71,16 +78,26 @@ const Navbar = (): React.ReactElement => {
...
       <Navigation.Actions>
+        <Navigation.LanguageSelector label={lang.toUpperCase()}>
+          <Navigation.Item
+            label="Suomeksi"
+            onClick={(): void => setLanguage('fi')}
+          />
+          <Navigation.Item
+            label="In English"
+            onClick={(): void => setLanguage('en')}
+          />
```
Comment (mhieta) [inline src/common/navbar/Navbar.tsx:89]: We should have placeholder for swedish UI as well.

File: src/i18n/en.json [hunk 4]
```diff
@@ -14,15 +29,17 @@
       "processingPersonalData": "Processing of personal data",
       "registryDescription": "Registry description",
       "support": "Support",
-      "title": "Urban parking permits service"
+      "title": "Urban parking permits",
+      "termsOfParkingCode": "Terms of the parking code",
```
Comment (mhieta) [inline src/i18n/en.json:33]: Should this be terms of "parking permit"? Or is the "parking code" something different?

**Review by mhieta — CHANGES_REQUESTED** [review]
@amanpdyadav Added some comments, but in overall looks very nice! 👍

File: src/i18n/en.json (hunk 4 above)
Comment (amanpdyadav) [reply src/i18n/en.json]: Done. Change to `Terms of the parking code`

File: src/common/navbar/Navbar.tsx (hunk 3 above)
Comment (amanpdyadav) [reply src/common/navbar/Navbar.tsx:93]: Done

File: src/i18n/fi.json (hunk 2 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Done

File: src/i18n/fi.json (hunk 1 above)
Comment (amanpdyadav) [reply src/i18n/fi.json]: Fixed

**Review by mhieta — APPROVED** [review]
Good, approved 💯

**Pull request merged**